### PR TITLE
Document taint-based eviction

### PR DESCRIPTION
**Pull request by sdudoladov**
based on #465

### CODE REVIEW

File: docs/administrator.md [hunk 1]
```diff
@@ -146,6 +146,11 @@ data:
   ...
 ```
 
+Note that the Kubernetes version 1.13 brings [taint-based eviction](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#taint-based-evictions) to the beta stage and enables it by default.
+Postgres pods by default receive tolerations for `unreachable` and `noExecute` taints with the timeout of `5m`.
+Depending on you setup, you may want to adjust these parameters to prevent master pods from being evicted by Kubernetes runtime.
```
Comment (FxKu) [inline docs/administrator.md]: your setup

Comment (FxKu) [reply docs/administrator.md]: by the Kubernetes runtime

File: docs/administrator.md [hunk 2]
```diff
@@ -146,6 +146,11 @@ data:
...
+To prevent eviction completely, specify the toleration without specifying the `tolerationSeconds` value (similar to how Kubernetes own DaemonSets are configured)
```
Comment (FxKu) [inline docs/administrator.md]: specify the toleration by leaving out the ...

Kubernetes' own ...

**Comment by sdudoladov** [comment]
@FxKu I updated the PR. Thanks for the review !

**Pull request merged**